src/modules/ordersTable/pure/OrdersTableContainer/index.tsx

```diff
@@ -210,7 +210,7 @@ export function OrdersTableContainer({
             ) : (
               <Trans>
                 You don&apos;t have any {isOpenOrdersTab ? 'open' : ''} orders at the moment. <br />
-                Create one for free! {/* TODO: add link for Advanced orders also */}
+                Time to create a new one! {/* TODO: add link for Advanced orders also */}
```

Comment:
is the text changing for both order types?
Fine if it's, just checking

Reply:
Yes...